### PR TITLE
fix: predict inverted index resource usage more reasonably

### PR DESCRIPTION
**Pull request by longjiquan**
/kind improvement
issue: #31617

### CODE REVIEW

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut

File: configs/milvus.yaml [hunk 1]
```diff
@@ -358,6 +358,7 @@ queryNode:
       maxPendingTaskPerUser: 1024 # 50 by default, max pending task in scheduler per user.
   mmap:
     mmapEnabled: false # enable mmap global, if set true, will use mmap to load segment data
+  invertedIndexLoadPredictMemoryUsageFactor: 0.2
```
Comment (xiaofan-luan) [inline configs/milvus.yaml:361]: why the memory consumption is much lesser than disk?

is this related to dataset we evaluate? since index itself has strong compression ratio? This might not be really safe? we should we directly store how much memory const for index?

Comment (longjiquan) [reply configs/milvus.yaml]: No, just because all the index will be mmapped, the actual memory occupied by the inverted index is up to the OS.

Comment (longjiquan) [reply configs/milvus.yaml]: No, just because all the index will be mmapped, the actual memory occupied by the inverted index is up to the OS.

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut
